### ISSUE TITLE
Crash Reporting?

### ISSUE THREAD

**Issue by hobuobi** [Recidiviz/public-dashboard#148]
Should investigate client-side crash reporting for the public dashboard. A few options discussed briefly with Colin: 
- Set something up within Google Analytics
- Set up a job to regularly visit each page on the site / interact with everything to make sure nothing's broken

**Comment by jessex** [comment]
There's also this: https://firebase.google.com/docs/crashlytics

The frontend of the app is hosted in Firebase so that might be a drop-in solution, though I haven't done much research.

**Comment by jessex** [comment]
Good news! I have set this up with no code change. [Sentry](http://sentry.io/) is set up for both pulse-dashboard and public-dashboard, I have tested that client-side errors in the frontend wind up in Sentry successfully, and I have set up initial alerting rules, starting with just emailing myself. We'll expand and reconfigure functionality as we learn more, but this is good to go for now!

Let me know what you'd like me to do before we close this issue out, @hobuobi, whether that's ensuring Ian has access or anything else.
